docs/user-guide/errors.md

```diff
@@ -65,7 +65,7 @@ The CSS parser built into Stylelint (or the chosen [custom syntax](options.md#cu
 
 The construct-specific parsers are:
 
-- `postcss-media-query-parser`
+- `@csstools/media-query-list-parser`
```

Comment:
[ask] I think it's good to add a changelog entry for this replacement to notify people, especially plugin authors. Any thoughts?

E.g.

```md
Fixed: remove `postcss-media-query-parser` dependency
```

https://github.com/stylelint/stylelint/blob/26f9351d733effaf14b9b0aa50cee8fb6060fe96/CHANGELOG.md?plain=1#L22

Reply:
Makes sense! I've added this.